lib/services/local/sessiontracker.go

```diff
@@ -123,7 +123,15 @@ func (s *sessionTracker) GetActiveSessionTrackers(ctx context.Context) ([]types.
 			return nil, trace.Wrap(err)
 		}
 
-		sessions[i] = session
+		if session.Expiry().After(time.Now().UTC()) {
```

Comment:
nit: Store `time.Now().UTC()` before the loop and reuse it? The tiny difference shouldn't matter.